pkg/controller/certificates/signer/signer_test.go

```diff
@@ -130,7 +130,7 @@ func TestHandle(t *testing.T) {
 			signerName: "kubernetes.io/kube-apiserver-client",
 			commonName: "hello-world",
 			org:        []string{"some-org"},
-			usages:     []capi.KeyUsage{capi.UsageClientAuth, capi.UsageDigitalSignature, capi.UsageKeyEncipherment},
```

Comment:
leave the existing testcases alone and add a copy of the test cases without the UsageKeyEncipherment usage to make sure the controller accepts both old and new

Reply:
updated